utils/utils.go

```diff
@@ -71,8 +25,3 @@ func RoundUpDivideBig(a, b *big.Int) *big.Int {
 	res.Div(a, b)
 	return res
 }
-
-func IsValidEthereumAddress(address string) bool {
```

Comment:
This method is used by a library which imports eigensdk

Reply:
which library?

Reply:
@samlaf eigenlayer-cli - https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/create.go#L195